Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:2-onbuild
+MAINTAINER James Swanick "swanijam@gmail.com"
+
+RUN apt-get update -qq
+RUN apt-get install unzip -y
```

Comment:
Remove unnecessary libraries in this list such `postgresql-client`. My approach would be to remove everything and see if it runs and then add back only the required ones (such as `git-core` and `unzip`).